### ISSUE TITLE
Are these losses normal when I train in coco

### ISSUE THREAD

**Issue by xuyongxi** [Duankaiwen/CenterNet#100]
focal loss at iteration 30995:    7.344306945800781
pull loss at iteration 30995:     0.05117112770676613
push loss at iteration 30995:     0.10568550229072571
regr loss at iteration 30995:     0.21536898612976074
training loss at iteration 31000: 7.482662200927734
focal loss at iteration 31000:    7.119882583618164
pull loss at iteration 31000:     0.044373102486133575
push loss at iteration 31000:     0.1216396912932396
regr loss at iteration 31000:     0.19676724076271057
validation loss at iteration 31000: 8.355660438537598

**Comment by fuqichen1998** [comment]
I have the same question, could @Duankaiwen explain if it is normal?

**Comment by Duankaiwen** [comment]
Hi, the losses are normal. These losses converge slowly. BTW, what's your batch size? @ xuyongxi @fuqichen1998

**Comment by fuqichen1998** [comment]
Thanks! My batch size is 16 for CenterNet-52.

**Comment by JoJoliking** [comment]
@fuqichen1998 I have set batch size 16 for CenterNet52. And the learning rate is changed into 2.5e-5 in 1000 epoch. (Using the CenterNet-480000 for pretianed model). Have you  achieve the author's result?

**Comment by JoJoliking** [comment]
@Duankaiwen  Dear author, Can you provide me with the final loss of CenterNet-52? This will help me judge whether the model has converged
